README.md

```diff
@@ -37,7 +37,7 @@ The [1.0 migration guide](https://github.com/functional-streams-for-scala/fs2/bl
 summarizes the differences between 1.0 and 0.10. To get 1.0.x, add the following to your SBT build:
 
 ```
-// available for Scala 2.11, 2.12
+// available for Scala 2.11, 2.12, 2.13
 libraryDependencies += "co.fs2" %% "fs2-core" % "2.0.0" // For cats 1.5.0 and cats-effect 1.2.0
```

Comment:
This seems odd also. 2.0.0 depends on cats-effect 2.x, not 1.2.0. 

The "Where to get the latest version" section above also appears outdated.